src/main/xar-resources/data/docs/fn/xml-to-json.md

```diff
@@ -0,0 +1,41 @@
+# fn:xml-to-json() ([specification](https://www.w3.org/TR/xpath-functions-31/#func-xml-to-json"))
+handles most valid input according to specification. Notable exceptions follow.
+
+All (un-)escaping is delegated to com.fasterxml.jackson. All output is generated by jackson
```

Comment:
```suggestion
All output is generated by the `jackson` library and should be proper JSON. All (un-) escaping is delegated to `com.fasterxml.jackson`. 
```

Reply:
@duncdrum's suggestion looks good to me here. @agh2342 Could you please accept or reject the suggestion when you have the chance?